internal/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -1538,7 +1538,11 @@ func (mvcc *MVCCLevelDB) GC(startKey, endKey []byte, safePoint uint64) error {
 
 // DeleteRange implements the MVCCStore interface.
 func (mvcc *MVCCLevelDB) DeleteRange(startKey, endKey []byte) error {
-	return mvcc.doRawDeleteRange(codec.EncodeBytes(nil, startKey), codec.EncodeBytes(nil, endKey))
+	var end []byte
+	if len(endKey) > 0 {
+		end = codec.EncodeBytes(nil, endKey)
+	}
```

Comment:
Why not do it for start key too?

Reply:
It is innecessary. There will be no key located before `""` or `encode("")` so encode or not encode both are fine.